clang/lib/Sema/SemaChecking.cpp

```diff
@@ -8092,6 +8092,18 @@ void Sema::checkCall(NamedDecl *FDecl, const FunctionProtoType *Proto,
     diagnoseArgDependentDiagnoseIfAttrs(FD, ThisArg, Args, Loc);
 }
 
+void Sema::CheckConstrainedAuto(TypeSourceInfo *TS) {
+  if (getLangOpts().CPlusPlus20) {
```

Comment:
Does the check for `getLangOpts().CPlusPlus20` does anything here?

Reply:
No, it doesn't. Removed.